server/app/views/questiontypes/EmailQuestionFragment.html

```diff
@@ -2,8 +2,13 @@
   th:fragment="email(question, questionRendererParams)"
   th:with="emailQuestion=${question.createEmailQuestion()},
               questionId=${'id-' + #strings.randomAlphanumeric(8)},
-              inputId=${'id-' + #strings.randomAlphanumeric(8)}"
+              inputId=${'id-' + #strings.randomAlphanumeric(8)},
+              fieldErrors=${emailQuestion.getValidationErrors().get(emailQuestion.getEmailPath())},
```

Comment:
question (non-blocking):  Same as with the currency question.  Why do we need both field and question errors since this is a single-input question?

Reply:
fixed